docs/resources/marker_setting.md

```diff
@@ -0,0 +1,38 @@
+# Resource: honeycombio_marker_setting
+
+Creates a marker setting. For more information about marker settings, check out [Marker Settings API](https://docs.honeycomb.io/api/marker-settings/).
+
+## Example Usage
+
+```hcl
+variable "dataset" {
+  type = string
+}
+
+variable "color" {
+  type = string
+}
+
+resource "honeycombio_marker_setting" "markerSetting" {
+  type = "deploy"
+  color = "${var.color}"
+
+  dataset = var.dataset
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `dataset` - (Required) The dataset this marker setting is placed on.
+* `type` - (Required) The type of the marker setting, Honeycomb.io can display markers in different colors depending on their type.
+* `color` - (Optional) The color set for the marker.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - ID of the marker setting.
+* `created_at` - Timestamp when the marker setting was created.
+* `updated_at` - Timestamp when the marker setting was last modified.
```

Comment:
At the moment these are not exported as read only, computed values but I think they could (maybe should? :)) be. You can do this by adding them to the resource's schema similar to the [dataset resource](https://github.com/honeycombio/terraform-provider-honeycombio/blob/main/honeycombio/resource_dataset.go#L41-L51) and then [setting them during the read function](https://github.com/honeycombio/terraform-provider-honeycombio/blob/main/honeycombio/resource_dataset.go#L93-L94)